### ISSUE TITLE
Starting Notion when Notion is minimized to the tray icon bar leads to stuck Notion processes

### ISSUE THREAD

**Issue by Stormxy** [notion-enhancer/notion-enhancer#439]
**describe the bug**
Once I launched Notion and "closed" (minimized) it to the tray icon bar I cannot open it via the Taskbar shortcut or any shortcut (or the notion.exe executable itself) again, only via interacting with the tray icon or via the ctrl+shift+A keyboard shortcut.

Even worse, trying to reopen Notion this way by starting Notion again (shortcut or .exe) seems to create new Notion processes (apparently two on every try), which seem to be stuck, not really doing anything visible, but each taking up 30% of my CPU usage.
Hence, trying over and over quickly makes the whole system hardly useable due to these stuck processes clogging up the whole CPU.
The stuck processes even stick around, when the still working, first started Notion process is quit via the tray icon bar.

**expected behaviour**
I would expect the still running Notion minimized to the tray icon bar to just be reshown when running Notion.exe again, as most programs do.
In any case trying to rerun Notion should not lead to stuck notion processes with permanent CPU usage.

**screenshots**
Windows Taskmanager with stuck Notion processes taking up 100% of my CPU.
![grafik](https://user-images.githubusercontent.com/6562837/110236297-4c001480-7f35-11eb-931c-5c9afcd1bd1c.png)

**platform**
Windows 10 using Notion 2.10.23.7.59.4 Desktop 2.0.14 according to Notion's help menu with Notion Enhancer 0.10.2 patched according to the workaround of #437.

**Comment by a1eaiactaest** [comment]
Yeah, same here, try `notion-enhancer remove` for a while until this issue is fixed.

**Comment by Moyf** [comment]
Same here  : (

**Comment by SantiagoOrtegaBanegas** [comment]
Does anybody fix the bug ? i will try to fix this issue  anybody  co work with me in this ? message me!

**Comment by Ilirski** [comment]
Try [this](https://github.com/notion-enhancer/notion-enhancer/issues/437)

**Comment by Stormxy** [comment]
@Ilirski As stated, I already use Notion Enhancer patched according to the workaround of #437, which you referenced, so there is nothing else to try there.

@a1eaiactaest Instead of completly disabling or removing Notion Enhancer, one can in the meantime also just disable the malfunctioning "close window to the tray" option in the notion-enhancer core module's settings.
This way one can at least still take advantage of the other features.
